accepted/future-releases/extension-types/feature-specification.md

```diff
@@ -828,6 +829,9 @@ type)*.  Exhaustiveness analysis will treat such patterns as if they had
 been an object pattern matching the extension type erasure of `V` (defined
 below).
 
+An extension type `V` is always-exhaustive if and only if its instantiated 
+representation type is always-exhaustive.
+
```

Comment:
Should we say somewhere, maybe around line 916, that `extension type E(SomeType _) implements SealedType {}` does not count as one of the "immediate subtypes" of the sealed type. Either the representation type *is* the sealed type, in which case the extension type inherits the always-exhaust, the representation type is an immediate subtype of the sealed type, in which case it counts towards exhausting that immediate subtype, or it's a further subtype of one or more immediate subtypes, in which case it probably doesn't count towards exhausting the sealed type (unless the intermediate types are also sealed).

Reply:
Done!